src/tabset/tabset.ts

```diff
@@ -110,11 +110,16 @@ export class NgbTabset implements AfterContentChecked {
   @Input() type: 'tabs' | 'pills';
 
   /**
+   * The horizontal alignment of the nav with flexbox utilities. Can be one of 'start', 'center' or 'end'
+   */
+  @Input() alignment: 'start' | 'center' | 'end';
```

Comment:
I had to pick between justify, align and alignement , @pkozlowski-opensource  should we change the name to justify?